### DISCUSSION TITLE
Puan gözükmesi

### DISCUSSION THREAD

**Discussion by Herkurtau** [IBAX-io/go-ibax#491] [Ideas]
### Describe the bug

Tüzük kısmında puan gözüküyo ama normal retweet yerinde puanlar gozukmuyor...

### Reproduction

Tüzük kısmında puan gözüküyo ama normal retweet yerinde puanlar gozukmuyor...

### System Info

```shell
Google chrome
```

### Logs

```shell
Bu da yeniden twitera baglamaya ugrastiriyo insanlar puan kazanamadığını düşünüyor...
```

### Validations

- [X] #488
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ibaxio** [comment]
Please check the answer of #453
